wordpress-mobile/WordPress-Android
# Issue/9815 domain credit cta

Fixes #9815

This PR picks up #9839 and adds couple of changes.

This PR adds a prompt to redeem domain credit in a form of CTA. 

[![Image from Gyazo](https://i.gyazo.com/eabbd7a52b7dd6577ea3c5f5becfaf98.png)](https://gyazo.com/eabbd7a52b7dd6577ea3c5f5becfaf98)



To test 1: 
- Install a Wasabi build.
- Select a site on Free plan.
- Navigate to MySite fragment.
- Notice domain registration CTA is not visible.

To test 2: 
- Install a Wasabi build.
- Select a site on any paid plan, without custom domain and unredeemed domain credit.
- Navigate to MySite fragment.
- Notice that Domain Registration CTA appears.
- Tap on CTA and registered domain name.
- Navigate back to MySite and notice that CTA is gone (it takes some time for a new domain name to get reflected on MySiteFragment).

Update release notes:
